sql/parser/normalize.go

```diff
@@ -318,11 +318,10 @@ func (expr *RangeCond) normalize(v *normalizeVisitor) TypedExpr {
 	)
 }
 
-func (expr *Row) normalize(v *normalizeVisitor) TypedExpr {
-	return &Tuple{
-		Exprs: expr.Exprs,
-		types: expr.types,
-	}
+func (expr *Tuple) normalize(v *normalizeVisitor) TypedExpr {
```

Comment:
@petermattis I retained this normalization step because we still claim to want `ROW(a, b, c) -> (a, b, c)`. Now that these are the same types, I'm not convinced that this is necessary as it just alters the string representation.